### ISSUE TITLE
Metabolic Syndrome Prediction | 4. ML-based Feature Importance

### ISSUE THREAD

**Issue by Arihant-Bhandari** [SrijanShovit/HealthLearning#89]
Step 4:

1. XGBoost (eXtreme Gradient Boosting model)
2. Random Forest
3. Decision Tree
4. LGBM (Light Gradient Boosting model) 
5. CatBoost (Categorical Boosting model)
6. Extra Trees
7. AdaBoost (Adaptive Boosting)
8. Gradient Boosting model

**Comment by Arihant-Bhandari** [comment]
@SrijanShovit hi sorry to disturb, i figured you were busy since you didnt create an issue for step 4. i have made this so that i can link up PR for the same, please mark this with appropriate labels, i have worked on these as is, i am sending a PR wherein you can review the work done for this. If there are things to be added or redone i will be more than glad to do so.

thank you for your time and patience.

additionally i have added a HistGradientBoostingClassifier-sklearn based permutation test for feature importance we covered in step 3 as an additon to this.
